src/Algebra/Construct/Add/Point.agda

```diff
@@ -0,0 +1,102 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definition of algebraic structures we get from adding a disjoint point
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Construct.Add.Point where
```

Comment:
You could in theory add a point element that was the zero instead of the identity element right? So this module should probably be called `Add.Identity` rather than `Add.Point` so that we have space for `Add.Zero` at some later date. You should also adjust the comment in the top header to match.